github-actions/check-data-assets/action.yml

```diff
@@ -72,7 +71,15 @@ runs:
           DEBUG=""
         fi
         export PYTHON_PATH="${{ inputs.python-path }}"
-        # If this was run on pull_request, comment on the PR if necessary
+
+        # Populated if we're running on: pull_request
+        PR_NUMBER=${{ github.event.number }}
```

Comment:
Would it be possible to provide the PR_NUMBER as an input parameter? That would cover also other cases when you are not explicitly running the check from a pull_request trigger, but as a workflow_dispatch or based on an issue comment.

E.g. including this in the inputs:
```
  pr-number:
    description: 'Pull request number'
    default: ${{ github.event.number }}
    required: false 
```
and then referring to it as `PR_NUMBER=${{ inputs.pr-number }}`

Reply:
Yes we can definitely do that, I'll coordinate with @mjperrone to either make that change in this PR, or get this merged and open a new one